lib/node_modules/@stdlib/math/base/special/bessely0/benchmark/r/benchmark.R

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env Rscript
+
+# Set the precision to 16 digits:
+options( digits = 16 );
+
+#' Run benchmarks.
+#'
+#' @examples
+#' main();
+main <- function() {
+	# Define benchmark parameters:
+	name <- "bessely0";
+	iterations <- 1000000L;
+	repeats <- 3;
+
+	#' Print the TAP version.
+	#'
+	#' @examples
+	#' print_version();
+	print_version <- function() {
+		cat( "TAP version 13\n" );
+	}
+
+	#' Print the TAP summary.
+	#'
+	#' @param total Total number of tests.
+	#' @param passing Total number of passing tests.
+	#'
+	#' @examples
+	#' print_summary( 3, 3 );
+	print_summary <- function( total, passing ) {
+		cat( "#\n" );
```

Comment:
Something happened to the indentation here.

Reply:
Ah, somehow you have mixed tabs and spaces for indentation. Make sure indentation is tabs in R files.

Reply:
If you have Editorconfig installed, this should be taken care of automatically via the config file in the root project directory.